test/blackbox-tests/test-cases/github1549/run.t

```diff
@@ -0,0 +1,29 @@
+Reproduction case for #1549: too many parentheses in installed .dune files
+
+  $ dune build --root backend
+  Entering directory 'backend'
+  $ dune install --root backend --prefix _install
```

Comment:
If you'd like to skip this install step, you can also just set a custom --build-dir and then point to its install subdir in the OCAMLPATH

Reply:
Indeed, or even just point to the default build directory. I did that